test/unit/mdc-textfield/mdc-text-field-bottom-line-foundation.test.js

```diff
@@ -53,7 +53,6 @@ test('#init adds event listeners', () => {
 test('#destroy removes event listeners', () => {
   const {foundation, mockAdapter} = setupTest();
   foundation.destroy();
-
```

Comment:
hahaha...just because you didn't like that empty line??

Reply:
hahah I thought there was a consistent pattern of where to put empty lines but there isn't! I'll just remove this change from this PR so it's less confusing